### PR TITLE
Update STaaS.md - Going Public

### PR DESCRIPTION
**Pull request by joaobarraca**

### CODE REVIEW

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 1]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
...
+
+  1. Restart your DMA and STaaS will be used.
+
+> [!IMPORTANT]
+>
+> Steps 4., 5. and 6. can only be done after registration is completed (Step 3.)
+
+> [!NOTE]
+>
+> If you experience any issues during setup or while using Storage as a Service do not hesitate to contact Cloud Ecosystem.
```
Comment (LouisSkyline) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:62]: There should be an official STaaS email I think, we should probably mention this here

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 2]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
...
+Migration to STaaS is supported for all datatypes with the exception of Logger Tables. Migration of Logger Tables will be supported in future versions. 
+
+> [!NOTE]
+>
+> Data stored in Cassandra and Elastic can only be migrated to STaaS if the databases are defined in the `DB.xml` configuration file.
+
+## Limitations
+
+- Jobs and Ticketing datatypes are not supported
+- No indexing engine functionality such as search and suggestions
```
Comment (LouisSkyline) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:74]: Should probably also add aliases and aggregation to this to be complete.

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 3]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
...
+## Migration
+Migration to STaaS is supported for all datatypes with the exception of Logger Tables. Migration of Logger Tables will be supported in future versions. 
+
+> [!NOTE]
+>
+> Data stored in Cassandra and Elastic can only be migrated to STaaS if the databases are defined in the `DB.xml` configuration file.
+
+## Limitations
+
+- Jobs and Ticketing datatypes are not supported
```
Comment (LouisSkyline) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:73]: Should you mention all unsupported features in one go? So also add API deployment to this list

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 4]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
 >
 > - To read more about Storage as a Service and its advantages, see [DataMiner STaaS: a game-changer for your storage needs](https://community.dataminer.services/dataminer-staas-a-game-changer-for-your-storage-needs/).
 > - As Storage as a Service is **scheduled for production in Q3 of 2023**, we are working on adding more information to this section. Check back for updates frequently!
+
+## Setup
+
+> [!IMPORTANT]
+>
+> Storage as a Service can be used from DataMiner version 10.3.10 (CU1) onwards.
```
Comment (LouisSkyline) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:39]: Should we mention that this is the minimum recommended version and that we always suggest installing the latest available version to get the latest features and performance updates?

**Review by LouisSkyline — CHANGES_REQUESTED** [review]

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md (hunk 1 above)
Comment (joaobarraca) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:62]: is there one?

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 5]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
...
+> We recommend the use of the latest available version to get the latest features and performance updates.
+
+  1. Make your system **Cloud Connected** and note down the organization and coordination ids. These can be found here: ClientTestTool>Advanced>CcaGateway>Get Global State. More information about getting cloud connected can be found [here](https://docs.dataminer.services/user-guide/Cloud_Platform/Connecting_to_cloud/Connecting_your_DataMiner_System_to_the_cloud.html).
+  1. Make sure you have at least **DataMiner CloudGateway 2.8.0** installed on the system. You can check this in the [Admin App](https://admin.dataminer.services) in the *nodes* section of your DMS. If this isn't the case, you can immediately update it from that page. You can find more information about this [here](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html)
+  1. Contact a Skyline representative to register you system to STaaS 
+  1. **Optional**: Migrate data to Cloud Storage. To migrate your current data to STaaS please contact your Skyline representative to do so.
```
Comment (GuillaumeVadeBe) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:45]:
```suggestion
  1. **Optional**: Migrate data to STaaS. To migrate your current data to STaaS please contact your Skyline representative to do so.
```

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 6]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
...
+
+  1. Restart your DMA and STaaS will be used.
+
+> [!IMPORTANT]
+>
+> Steps 4., 5. and 6. can only be done after registration is completed (Step 3.)
+
+> [!NOTE]
+>
+> If you experience any issues during setup or while using Storage as a Service do not hesitate to contact the Cloud Ecosystem, by sending an email to: staas@skyline.be.
```
Comment (GuillaumeVadeBe) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:63]:
```suggestion
> If you experience any issues during setup or while using Storage as a Service, do not hesitate to contact the Cloud Ecosystem, by sending an email to: staas@skyline.be.
```

**Review by GuillaumeVadeBe — CHANGES_REQUESTED** [review]

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RobbertVH) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:45]: or to "the Cloud"

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 7]
```diff
@@ -31,3 +31,69 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
...
+
+## Migration
+Migration to STaaS is supported for all datatypes with the exception of Logger Tables. Migration of Logger Tables will be supported in future versions. 
+
+> [!NOTE]
+>
+> Data stored in Cassandra and Elastic can only be migrated to STaaS if the databases are defined in the `DB.xml` configuration file.
```
Comment (RobbertVH) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:70]:
```suggestion
> Data stored in Cassandra and Elastic databases can only be migrated to STaaS if the databases are defined in the `DB.xml` configuration file.
```

**Review by LouisSkyline — APPROVED** [review]

**Review by RobbertVH — APPROVED** [review]

**Pull request merged**